brainstormforce/astra
# AST-878 - Editor BG white when full-screen mode is active

### Description
- Inline block editor CSS affects on content area background
- Added `!important` as this was inline CSS & needs to be override

### Screenshots
- https://share.bsf.io/DOuB470n

### Types of changes
- Bug fix

### How has this been tested?
- Open editor and set any layout 
- Check now with Full-Screen mode
- Can be check by zoom-in/out screen resolution

### Checklist:
